### PR TITLE
Separate open source from hosted supported databases in reference table

### PR DESCRIPTION
**Pull request by zachtil**
## Describe this PR

Add visual separation between open source and hosted databases in reference table.

## Changes

Changes to the Supported databases table in `03-supported-databases.mdx`

## What issue does this fix?

n/a

### CODE REVIEW

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 1]
```diff
@@ -10,25 +10,20 @@ Prisma currently supports the following databases:
...
-| PostgreSQL                   | 10      |
-| PostgreSQL                   | 11      |
-| PostgreSQL                   | 12      |
-| PostgreSQL                   | 13      |
-| PostgreSQL                   | 14      |
-| MySQL                        | 5.6     |
-| MySQL                        | 5.7     |
-| MySQL                        | 8       |
-| MariaDB                      | 10      |
+| _**Self-hosted databases**_  |         |
```
Comment (janpio) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion
```

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 2]
```diff
@@ -10,25 +10,20 @@ Prisma currently supports the following databases:
...
-| AWS Aurora Serverless &sup1; | \*      |
-| Microsoft SQL Server         | 2022    |
-| Microsoft SQL Server         | 2019    |
-| Microsoft SQL Server         | 2017    |
-| Azure SQL                    | \*      |
 | MongoDB                      | 4.2+    |
+| Microsoft SQL Server         | 2022<br />2019<br />2017 |   
 | CockroachDB                  | 21.2.4+ |
+| MariaDB                      | 10      |
+| _**Cloud databases (DBaaS)**_|         |
```
Comment (janpio) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion
| _**Managed Databases**_|         |
```

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 3]
```diff
@@ -10,25 +10,20 @@ Prisma currently supports the following databases:
...
+| AWS Aurora                   | \*      |
+| AWS Aurora Serverless        | \* &sup1;     |
+| Azure SQL                    | \*      |
+| CockroachDB                  | \*      |
```
Comment (janpio) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion
| CockroachDB-as-a-Service     | \*      |
```

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 4]
```diff
@@ -10,25 +10,19 @@ Prisma currently supports the following databases:
...
-| Azure SQL                    | \*      |
 | MongoDB                      | 4.2+    |
+| Microsoft SQL Server         | 2022<br />2019<br />2017 |   
 | CockroachDB                  | 21.2.4+ |
+| MariaDB                      | 10      |
+| _**Managed Databases**_|         |
+| AWS Aurora                   | \*      |
+| AWS Aurora Serverless        | \* &sup1;     |
+| Azure SQL                    | \*      |
+| CockroachDB-as-a-Service | \*      |
```
Comment (janpio) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion
| _**Managed database**_      |         |
| AWS Aurora                   | \*      |
| AWS Aurora Serverless        | \* &sup1;     |
| Azure SQL                    | \*      |
| CockroachDB-as-a-Service     | \*      |
```

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 5]
```diff
@@ -10,25 +10,19 @@ Prisma currently supports the following databases:
...
-| PostgreSQL                   | 11      |
-| PostgreSQL                   | 12      |
-| PostgreSQL                   | 13      |
-| PostgreSQL                   | 14      |
-| MySQL                        | 5.6     |
-| MySQL                        | 5.7     |
-| MySQL                        | 8       |
-| MariaDB                      | 10      |
+| PostgreSQL                   | 9.4+<br />10<br />11<br />12<br />13<br />14 |
+| MySQL                        | 5.6+<br />5.7<br />8 |
```
Comment (janpio) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion
| MySQL                        | 5.6<br />5.7<br />8 |
```

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 6]
```diff
@@ -10,25 +10,19 @@ Prisma currently supports the following databases:
...
 | SQLite                       | \*      |
-| AWS Aurora                   | \*      |
-| AWS Aurora Serverless &sup1; | \*      |
-| Microsoft SQL Server         | 2022    |
-| Microsoft SQL Server         | 2019    |
-| Microsoft SQL Server         | 2017    |
-| Azure SQL                    | \*      |
 | MongoDB                      | 4.2+    |
+| Microsoft SQL Server         | 2022<br />2019<br />2017 |
```
Comment (janpio) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion
| Microsoft SQL Server         | 2017<br />2019<br />2022 |   
```

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 7]
```diff
@@ -10,25 +10,19 @@ Prisma currently supports the following databases:
...
-| AWS Aurora Serverless &sup1; | \*      |
-| Microsoft SQL Server         | 2022    |
-| Microsoft SQL Server         | 2019    |
-| Microsoft SQL Server         | 2017    |
-| Azure SQL                    | \*      |
 | MongoDB                      | 4.2+    |
+| Microsoft SQL Server         | 2017<br />2019<br />2022 |   
 | CockroachDB                  | 21.2.4+ |
+| MariaDB                      | 10      |
+| _**Managed database**_      |         |
```
Comment (janpio) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion
| _Managed database_          |         |
```

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 8]
```diff
@@ -21,16 +21,19 @@ Prisma currently supports the following databases:
...
-| AWS Aurora Serverless &sup1; | \*      |
-| Microsoft SQL Server         | 2022    |
-| Microsoft SQL Server         | 2019    |
 | Microsoft SQL Server         | 2017    |
-| Azure SQL                    | \*      |
+| Microsoft SQL Server         | 2019    |
+| Microsoft SQL Server         | 2022    |
 | MongoDB                      | 4.2+    |
 | CockroachDB                  | 21.2.4+ |
+| _Managed database_           |         |
```
Comment (nikolasburk) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion

## Managed databases

```

**Comment by nikolasburk** [comment]
I'm suggesting to make the visual separation of managed databases with an extra hading and using a second table.

**Pull request merged**